### ISSUE TITLE
Docker compose does not work

### ISSUE THREAD

**Issue by ilyaglow** [google/timesketch#333]
Simple `docker-compose up` won't succeed because of the `docker-entrypoint.sh` permissions issue:

```
ERROR: for timesketch  Cannot start service timesketch: oci runtime error: container_linux.go:247: starting container process caused "exec: \"/docker-entrypoint.sh\": permission denied"

ERROR: Encountered errors while bringing up the project.
```

I solved this within #329 which contains a couple more small fixes related to `Dockerfile` and `docker-compose.yml`

Sorry for making things backwards, I should have opened an issue first.

**Comment by berggren** [comment]
No worries, thanks for fixing this! I left some comment on the PR.

**Comment by ilyaglow** [comment]
Solved by #354
